jenkins-scripts/docker/ign_gazebo-compilation.bash

```diff
@@ -29,18 +29,17 @@ if ! [[ ${IGN_GAZEBO_MAJOR_VERSION} =~ ^-?[0-9]+$ ]]; then
 fi
 
 if [[ ${IGN_GAZEBO_MAJOR_VERSION} -ge 4 ]]; then
+  export BUILD_IGN_PHYSICS=true
+  export IGN_PHYSICS_MAJOR_VERSION=4
+  export IGN_PHYSICS_BRANCH=master
+
   export BUILD_IGN_SENSORS=true
   export IGN_SENSORS_MAJOR_VERSION=4
   export IGN_SENSORS_BRANCH=master
+
   export BUILD_IGN_GUI=true
   export IGN_GUI_MAJOR_VERSION=4
   export IGN_GUI_BRANCH=master
-  export BUILD_IGN_RENDERING=true
-  export IGN_RENDERING_MAJOR_VERSION=4
-  export IGN_RENDERING_BRANCH=master
```

Comment:
We can't remove `rendering4`, that's the current `master` branch, that will be in Dome

Reply:
ouch, 65d396f60563c0c13bf2bcb07372e950c93b449a